DB/Pets/Dragonflight.lua

```diff
@@ -161,6 +161,21 @@ local dragonflightPets = {
 		equalOdds = true,
 		coords = { { m = CONSTANTS.UIMAPIDS.THE_FORBIDDEN_REACH, x = 55.8, y = 51.6, n = L["Luttrok"] } },
 	},
+	["Emmah"] = {
+		cat = CONSTANTS.ITEM_CATEGORIES.DRAGONFLIGHT,
+		type = CONSTANTS.ITEM_TYPES.PET,
+		method = CONSTANTS.DETECTION_METHODS.SPECIAL,
+		name = L["Emmah"],
+		zones = { "2151" },
+		spellId = 375511,
+		itemId = 193853,
+		creatureId = 189694,
+		sourceText = L["Fished from Disgusting Vat located within the Zskera Vault"],
+		requiresPool = true,
```

Comment:
If this item requires a pool, shouldn't the detection logic check for `Rarity.isPool`?

Reply:
Answer: It's not needed here, since we don't rely on the builtin fishing method (see Mr. Pinchy).